pkg/BUILD

```diff
@@ -58,11 +58,10 @@ py_binary(
     srcs = ["make_rpm.py"],
     python_version = "PY3",
     srcs_version = "PY3",
-    # TODO(aiuto): Enable this for bazel 4.x
-    #target_compatible_with = select({
-    #    "//toolchains:have_rpmbuild": [],
-    #    "//conditions:default": [":not_compatible"],
-    #}),
+    target_compatible_with = select({
+        "//toolchains:have_rpmbuild": [],
+        "//conditions:default": [":not_compatible"],
+    }),
```

Comment:
This doesn't _quite_ do what we want it to, but it's a definite improvement.  Without a local copy of `rpmbuild`, I get a bunch of skipped tests, but I get this for a few of them that depend on the outputs of the action that calls `rpmbuild`:

```console
> USE_BAZEL_VERSION=rolling  bazelisk test //tests/rpm:analysis_tests_naming_with_different_name_rpm
DEBUG: /private/var/tmp/_bazel_apsaltis/297e6fd0ec15ec7901f2a2bf25d17005/external/rules_pkg/toolchains/rpmbuild_configure.bzl:35:16: No system rpmbuild found.
DEBUG: /private/var/tmp/_bazel_apsaltis/297e6fd0ec15ec7901f2a2bf25d17005/external/rules_pkg/toolchains/git/git_configure.bzl:34:18: Found git at '/opt/local/bin/git'
DEBUG: /private/var/tmp/_bazel_apsaltis/297e6fd0ec15ec7901f2a2bf25d17005/external/rules_pkg/toolchains/git/git_configure.bzl:42:14: Found WORKSPACE in /Users/apsaltis/workspace-git/rules_pkg/pkg
ERROR: Target //tests/rpm:analysis_tests_naming_with_different_name_rpm is incompatible and cannot be built, but was explicitly requested.
Dependency chain:
    //tests/rpm:analysis_tests_naming_with_different_name_rpm
    //:make_rpm   <-- target platform (@local_config_platform//:host) didn't satisfy constraint //:not_compatible
INFO: Elapsed time: 14.556s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (50 packages loaded, 645 targets configured)
```